system/lib/wasmfs/special_files.cpp

```diff
@@ -130,4 +156,14 @@ std::shared_ptr<DataFile> getStderr() {
   return stderr;
 }
 
+std::shared_ptr<DataFile> getRandom() {
+  static auto random = std::make_shared<RandomFile>();
+  return random;
+}
+
+std::shared_ptr<DataFile> getURandom() {
+  static auto urandom = std::make_shared<RandomFile>();
+  return urandom;
+}
```

Comment:
Side note, eventually we'll want to think about code size here, as this will get included in all builds, including the JS support. Maybe we could add a "full POSIX filesystem support" flag of some kind.

Reply:
Yeah, that sounds reasonable to me.